src/imitation/scripts/train_imitation.py

```diff
@@ -22,13 +22,16 @@ def make_policy(
     venv: vec_env.VecEnv,
     policy_cls: Type[policies.BasePolicy],
     policy_kwargs: Mapping[str, Any],
+    policy_path: Optional[str],
 ) -> policies.BasePolicy:
     """Makes policy.
 
     Args:
         venv: Vectorized environment we will be imitating demos from.
         policy_cls: Type of a Stable Baselines3 policy architecture.
         policy_kwargs: Keyword arguments for policy constructor.
+        policy_path: Path to serialized policy. If provided, then load the
```

Comment:
If this is specified then `policy_cls` and `policy_kwargs` are ignored. Worth either documenting, or adding input validation to check that `policy_{cls,kwargs}` match the deserialized policy.

Reply:
Not sure if all the policy kwargs are always stored as attributes in the class that can be checked by looping and doing `getattr`. I think it's not obvious that incompatible kwargs should raise an error rather than replace the ones from the saved policy, so we should probably not support it and raise a ValueError when both are provided